packages/plugin-ext/src/plugin/types-impl.ts

```diff
@@ -423,7 +423,7 @@ export class Position {
         return false;
     }
 
-    toJSON(): any {
+    toJSON(): unknown {
```

Comment:
`object` would make more sense here.

```suggestion
    toJSON(): object {
```

IIUC the original author didn't want to bother declaring the object being returned, hence the `any`, but using `unknown` instead feels odd.